packages/theme_tailor/README.md

```diff
@@ -240,7 +241,31 @@ class NotGeneratedExtension extends ThemeExtension<NotGeneratedExtension> {
 }
 ```
 
-To see examle implementation of nested theme, head out to: [example:nested_themes][example: nested_themes]
+To see example implementation of nested theme, head out to: [example:nested_themes][example: nested_themes]
```

Comment:
Current markdown tag is inappropriate, as display only text, not link to example, so could be fixed by the way (without additional PR)
```suggestion
To see example implementation of nested theme, head out to: [example: nested_themes][example:nested_themes]
```